packages/timeline-state-resolver/src/integrations/tricaster/externalStateConverter.ts

```diff
@@ -0,0 +1,160 @@
+import {
+	TriCasterAudioChannelName,
+	TriCasterDelegateName,
+	TriCasterInputName,
+	TriCasterKeyerName,
+	TriCasterLayerName,
+	TriCasterMixEffectName,
+	TriCasterMixOutputName,
+} from 'timeline-state-resolver-types'
+import { ElementCompact, xml2js } from 'xml-js'
+import {
+	fillRecord,
+	TriCasterMixEffectState,
+	TriCasterAudioChannelState,
+	TriCasterInputState,
+	TriCasterKeyerState,
+	TriCasterLayerState,
+	TriCasterMixOutputState,
+	TriCasterState,
+} from './triCasterStateDiffer'
+import { CommandName, TriCasterGenericCommandName } from './triCasterCommands'
+
+type ShortcutStates = {
+	[key: string]: string | number
+}
+
+export class ExternalStateConverter {
```

Comment:
This name is broad. 
First of the implementation focusses on Tricaster, so that should atleast be included in the name. (if you import somewhere not Tricaster related, you would have no idea that this converted state for Tricaster)
If possible, it would also be much appreciated if the name could include what it converted from and to (external state could be anything)